### PR TITLE
Addressing early review comments from Marcos.

### PR DESCRIPTION
**Pull request by emtwo**

### CODE REVIEW

File: js/rect.js [hunk 1]
```diff
@@ -22,13 +22,13 @@
         return this.bottom - this.top;
       },
       set width(v) {
-        this.right = this.left + v;
+        this.right = this.left + Number(v);
```
Comment (marcoscaceres) [inline js/rect.js]: How about: 

``` JS
var num = Number.parseInt(v, 10);
Number.isNaN(num){
  return;
}
this.right = this.left + num;
```

File: js/rect.js [hunk 2]
```diff
@@ -22,13 +22,13 @@
...
       },
       set height(v) {
-        this.bottom = this.top + v;
+        this.bottom = this.top + Number(v);
```
Comment (marcoscaceres) [inline js/rect.js]: As above.

File: js/rect.js [hunk 3]
```diff
@@ -22,13 +22,13 @@
...
       },
 
-      isEmpty() {
+      get isEmpty() {
         return this.left >= this.right || this.top >= this.bottom;
```
Comment (marcoscaceres) [inline js/rect.js:40]: :+1:

File: js/rect.js (hunk 1 above)
Comment (marcoscaceres) [inline js/rect.js]: I wonder if we should have a generic type enforcement for Rect that throws if any of the values is NaN. Also, there is already a Rect class on the Window object it seems (that is different from this one).

Comment (emtwo) [inline js/rect.js]: I'll make the change in your first comment and file an issue for generic type enforcement and changing the name of Rect to something else. How does that sound?

**Comment by marcoscaceres** [comment]
r+

**Pull request merged**